src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -585,6 +585,11 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 TextSpan,
                 "BCP105",
                 $"Unable to load file from URI \"{fileUri}\".");
+            
+            public ErrorDiagnostic CommaSeparator() => new ErrorDiagnostic(
+                TextSpan,
+                "BCP106",
+                "Commas are not used as separator delimiters. Use new lines");

```

Comment:
You should use the expected message token as the prefix since it's just providing additional information on top of that.

Reply:
Same for the name of the method here.

Reply:
Made the appropriate changes